crates/prover/src/components/opcodes/u32_store_bitwise_fp_fp.rs

```diff
@@ -0,0 +1,831 @@
+//! This component is used to prove the U32StoreBitwiseFpFp opcode.
+//! u32([fp + dst_off], [fp + dst_off + 1]) = u32([fp + src0_off], [fp + src0_off + 1]) &/|/^ u32([fp + src1_off], [fp + src1_off + 1])
+//!
+//! # Columns
+//!
+//! - enabler
+//! - pc
+//! - fp
+//! - clock
+//! - inst_prev_clock
+//! - opcode_constant
+//! - src0_off
+//! - src1_off
+//! - dst_off
+//! - op0_val_0
+//! - op0_val_1
+//! - op0_val_2
+//! - op0_val_3
+//! - op0_prev_clock_lo
+//! - op0_prev_clock_hi
+//! - op1_val_0
+//! - op1_val_1
+//! - op1_val_2
+//! - op1_val_3
+//! - op1_prev_clock_lo
+//! - op1_prev_clock_hi
+//! - dst_prev_val_lo
+//! - dst_prev_val_hi
+//! - res_0
+//! - res_1
+//! - res_2
+//! - res_3
+//! - dst_prev_clock_lo
+//! - dst_prev_clock_hi
+//!
+//! # Constraints
+//!
+//! * enabler is a bool
+//!   * `enabler * (1 - enabler)`
+//! * opcode_constant is 36, 37 or 38
+//!   * `(36 - opcode_constant) * (37 - opcode_constant) * (38 - opcode_constant)`
+//! * define `bitwise_op = opcode_constant - 36`
```

Comment:
it is really necessary to add the constraint on the opcode_constant - 36/37/38 ? i mean that you can just extract the bitwise_op = opcode_constant - 36 and if it's not a valid value the lookup will fail